src/benchmarks/micro/corefx/System.Text.Json/Utf8JsonWriter/Perf.HelloWorld.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers;
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
+namespace System.Text.Json
+{
+    [BenchmarkCategory(Categories.CoreFX, Categories.JSON)]
+    public class Perf_HelloWorld
+    {
+        private static readonly byte[] Message = Encoding.UTF8.GetBytes("message");
+        private static readonly byte[] HelloWorld = Encoding.UTF8.GetBytes("Hello, World!");
```

Comment:
If we keep this file perhaps
```suggestion
        private static readonly byte[] HelloWorldUtf8 = Encoding.UTF8.GetBytes("Hello, World!");
```